0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -1,252 +1,135 @@
 # Interledger Architecture
 
-This document outlines the Interledger architecture and explains how the different layers relate to each other. The [Interledger Protocol (ILP)](#interledger-protocol-ilp) is one layer in the Interledger architecture.
+The Interledger protocol suite provides for secure payments across multiple assets on different ledgers.
```

Comment:
I like the usage of these grouping terms like `Interledger Architecture` and `Interledger Protocol Suite`, but I'm not sure the document clearly differentiates between the two terms.  Are these meant to be synonymous, or is “Interledger Protocol Suite” a component of the “Interledger Architecture”, which actually has more to it than just the protocol suite?